src/plugins/ganttChart/ganttChart.js

```diff
@@ -12,9 +12,8 @@ import './ganttChart.css';
 
 /**
  * @plugin GanttChart
- * @experimental
  * @dependencies CollapsibleColumns
- *
+ * @deprecated This plugin is deprecated and will be deleted soon
```

Comment:
I'm not sure if we should use the word "soon". It may take some time. "with the next major release"? But I'm not sure either. We may keep it for a little longer. As long as it works correctly why not.

```suggestion
 * @deprecated This plugin is deprecated and will be removed in the next major release
```

Sounds good 🤔 I think... If decide to keep it longer the message will still be relevant

Reply:
Have you checked if `@deprecated` is displayed in the docs @scarletfog? If not we may have to ask @budnix for help

Reply:
AFAIR, it was up and running locally but yes, a double-check by @budnix will be much appreciated!

Reply:
I've checked and the `@deprecated` tag is displayed in the docs but it seems that it is unstyled:
![Zrzut ekranu 2020-01-7 o 10 40 20](https://user-images.githubusercontent.com/571316/71885282-527cdb80-313a-11ea-8a79-c5ff77cd1af8.png)
Another matter is the fact that the deprecation warning/notice is visible only in the constructor description (Basic usage). Is this enough?

Reply:
🤔 I am almost sure it was working on https://github.com/handsontable/docs/tree/draft-8.0.0 I will check it

Reply:
I've checked codebase from 7.2.2. I'll check draft 8.0.0.

Reply:
Yep, on the draft version, the message is correctly styled 
![Zrzut ekranu 2020-01-7 o 10 53 12](https://user-images.githubusercontent.com/571316/71886108-f4e98e80-313b-11ea-9837-5e6ccdcc9ffa.png) 👍

Reply:
@wojciechczerniak I just committed the suggestion, thanks! I will also update it in the docs.